lib/_http_server.js

```diff
@@ -488,6 +505,16 @@ Server.prototype[EE.captureRejectionSymbol] = function(err, event, ...args) {
   }
 };
 
+function connectionsToResource(resources) {
+  const connections = Array(resources.length);
```

Comment:
Wouldn't it be better to use a `Set`?

Reply:
It might but in the reality this `Array` is actually a set (the underlying data structure in C++ is a `std::set`) so it doesn't make a real difference.

Reply:
Checking whether a socket is in the collection would take `O(n)` with an array.

Reply:
🤦‍♂️ 

You are right, I was not thinking about developer-side search. Ok, will change to be a `Set`.